node/utils/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const STATUS = {
+  ACTIVE: 'active',
+  APPROVED: 'approved',
+  DECLINED: 'declined',
+  PENDING: 'pending',
+  SUCCESS: 'success',
+}
```

Comment:
I suggest separating this into two constants, one for Organization Request statuses (PENDING, APPROVED, DECLINED) and one for Organization statuses (ACTIVE, INACTIVE, ON-HOLD)

Then for example in the `updateOrganization` resolver we could check if the provided status argument is a valid value